README.md

```diff
@@ -10,22 +10,28 @@ The dependency list is short: numpy and Cython (>= 0.21.1).
 Usage
```

Comment:
And also:
* the tokenizer
* anything a user needs to know about encoding, especially if they are implementing their own tokenizer